docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Managing Pull Requests
+---
+
+## Pull Requests and the Community
+
+Gatsby is an open source project. We have over 2,000 contributors and so much of what makes Gatsby great is contributed by folks like you!
+
+Needless to say, we get a lot of PRs and we've been merging over a [100 contributions](https://twitter.com/kylemathews/status/1111435640581689345) every week. Yes, _every week_.
+
+Let's talk a little about how we manage pull requests in the Gatsby repo.
+
+## What is a Pull Request?
+
+In case you aren't familiar, here is how the fine folks at GitHub [define a pull request](https://help.github.com/en/articles/about-pull-requests):
+
+> Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.
+
+Simply put, when someone wants to contribute to Gatsby, they open a request to _pull_ their code into ours. Depending on the type of change, we like to categorise pull requests (or PRs for short) into:
+
+- Documentation
+- Code
+- Starters or Site Showcase
+- Blog posts
+
+When looking at a PR for the first time, it can help to read up on linked issues or RFCs (if there are any) to gain context on what the PR intends to add or fix.
+
+> 💡 If there isn't already an issue that describes the problem a PR is trying to solve, it can be helpful to open one. This helps in sharing context and can be very valuable in the future.
+
+## Verifying a Pull Request
+
+### General Guidelines
+
+Some general things to verify in a pull request are:
+
+- Links ought to be relative
```

Comment:
```suggestion
- Links ought to be relative instead of absolute when linking to docs (`/docs/some-reference` instead of `https://www.gatsbyjs.org/docs/some-reference`)
```

Not sure there would be another use case besides linking to docs, but I think a little example could help 🙂